extendr-api/src/wrapper/function.rs

```diff
@@ -29,6 +29,7 @@ pub struct Function {
 }
 
 impl Function {
+    #[cfg(feature = "non-api")]
```

Comment:
Note that this is a breaking change. Goes in changelog.

Reply:
@CGMossa thoughts on this? I'm fine removing it for now. I don't think it is being used in the wild at the moment.

Reply:
I'm for removing these for now.